cmd/krew/cmd/install.go

```diff
@@ -94,11 +102,14 @@ Remarks:
 				install = append(install, plugin)
 			}
 
-			if *manifest != "" {
-				plugin, err := indexscanner.ReadPluginFile(*manifest)
+			var plugin index.Plugin
+			if *manifest != "" || *manifestURL != "" {
+				var err error
+				plugin, err = internal.GetPlugin(*manifest, *manifestURL)
 				if err != nil {
 					return errors.Wrap(err, "failed to load custom manifest file")
 				}
+
 				if err := validation.ValidatePlugin(plugin.Name, plugin); err != nil {
```

Comment:
I would have moved that into `GetPlugin` as well, but I guess it's ok.

Reply:
I thought writing these separately will be able to better differentiate between `loading manifest` and `validation of manifest` errors